document-en.md

```diff
@@ -0,0 +1,244 @@
+## FramgiaCI Documents
+`Sun*CI` is a CI/CD system developed by Sun* Asterisk , beside the main features like the other CI/CD system, `Sun*CI` provided more features for assisting user in developing projects.Here's some  instruction you better look before intergrate with `Sun*CI`
+## Permission
+Currently, `Sun*CI` is suporting user login to system using github, gitlab and bitbucket account, after login, by default your permission will be `User`. Your accessibility  to `Sun*CI` base on your permission as show below:
```

Comment:
`github, gitlab and bitbucket` => `Github, Gitlab (both https://gitlab.com/ and Sun* Gitlab), Bitbucket`

`after login` => `after logging in`